packages/grainIntegration-merkle/README.md

```diff
@@ -0,0 +1,42 @@
+# Merkle Distribution Grain Integration
+
+#### ⚠️ This contract is still under active development ⚠️
+
+## How It Works
+
+`MerkleReem` is heavily based on [work from balancer][balancer]
+and is a scalable way to distribute rewards in the form of an ERC20 on an EVM
+blockchain. There are 3 roles:
+
+- `DEFAULT_ADMIN`: Assigned to the deployer of the contract. This role
+  is responsible for managing the below two roles
+- `SEEDER_ROLE`: Manually assigned and managed by the `DEFAULT_ADMIN`
+  role. Responsible for publishing distributions on chain using the
+  `seedAllocations` function.
+- `PAUSER_ROLE`: Manually assigned and managed by the `DEFAULT_ADMIN`
+  role. Responsible for pausing a distribution, which can be unpaused.
```

Comment:
Might be good to note here that pausing happens by removal, and that the pauser is also responsible for the minimum delay.